### DISCUSSION TITLE
practice

### DISCUSSION THREAD

**Discussion by Nandani-chauhan** [Jay-study-nildana/FrontEndForStudents#35] [General]
Hi sir,
i am attending your lecture  through to Pre-recorded classes from edureka . And i practice to using git hub and how to ask my query .
these pre-recorded session are very useful to us and i am learning a lot. thank you very much .

**Comment by Jay-study-nildana** [comment]
@Nandani-chauhan 

Hello and thank you for your kind words. I would strongly recommend you check out these 'beginner videos' I made.

https://www.youtube.com/playlist?list=PL7SHzFbuSdW0OLny_GyfMxLKidxeurrPz 

And, I provide free and paid internships to students. 

Free Internship: https://github.com/BigbabeeSmallbabee 

Paid Internship: (For Bangalore based students only) https://medium.com/projectwt/hiring-javascript-and-c-sharp-interns-for-coding-workshop-business-755cb4d617f8 

Happy Coding and any questions, please feel free to get in touch with me. 

Thanks

Jay (Vijayasimha BR)
Coding Tutor. Career and Life Coach
----
Discord: Jay_HisManyHobbies#6392
